account_mass_reconcile/tests/test_scenario_reconcile.py

```diff
@@ -213,6 +213,9 @@ def test_reconcile_with_writeoff(self):
                 ],
             }
         )
+        # mass_rec.flush_recordset()
+        # invoice.payment_id = payment.id
```

Comment:
@sonhd91 ty, btw why do we need to comment this code?

Reply:
Sorry my bad, I just update the branch, please check again